clang/test/Driver/fveclib.c

```diff
@@ -31,3 +31,27 @@
 
 // RUN: %clang -fveclib=Accelerate %s -nodefaultlibs -target arm64-apple-ios8.0.0 -### 2>&1 | FileCheck --check-prefix=CHECK-LINK-NODEFAULTLIBS %s
 // CHECK-LINK-NODEFAULTLIBS-NOT: "-framework" "Accelerate"
+
+
+/* Verify that the correct vector library is passed to LTO flags. */
+
+// RUN: %clang -### -fveclib=Accelerate -flto %s -v 2>&1 | FileCheck -check-prefix CHECK-LTO-ACCELERATE %s
```

Comment:
you are missing here target, presumably it should be  -target arm64-apple-ios8.0.0 as in the other run lines for "-fveclib=Accelerate", I think you need to ensure that each of the RUN lines specifies target

Reply:
Actually it turns out that some of the veclibs (like Accelerate in this case) do not have any target/toolchain that would call `addLTOOptions`. So I removed those.

The ones that remain, all specify a target.